### PR TITLE
Add navbar component

### PR DESCRIPTION
**Pull request by DavideSilva**
Why:
* It's a part of the UI components needed to build the page

These changes address the issue by:
* Adding a Navbar component
* Adding a NavLink component
* Adding a NavButton component
* Sorting the imports in the kitchensink
* Adding the Navbar to the kitchensink
* Adding padding top to the Layout component so that we can see the content
* Adding the Navbar to the index

### CODE REVIEW

**Comment by DavideSilva** [comment]
The logo size change on scroll is still missing.

Also, the NavLink and NavButton components are copy-pasted from the Link and Button, respectively. The only diferences are related to styling.

And can we try to keep the imports sorted alphabetically? Not a major issue but I think that it helps code readability and also it helps when there are rebasing conflicts.

**Comment by DavideSilva** [comment]
Oh and also, please go to town reviewing this. I'm not very proficient at this frontend stuff, so please feel free to show me how it should be done 😅

File: src/components/Navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import Logo from "root/components/Logo";
+import NavLink from "root/components/NavLink";
+import NavButton from "root/components/NavButton";
+
+import styles from "./index.module.css";
+
+const Navbar = () => {
+  return (
+    <header className={styles.root}>
```
Comment (jfranciscosousa) [inline src/components/Navbar/index.js]: `header` has a different meaning, it's used to specify the header of a section. `h1` and so on do that automatically. No need to use this. The important tag here is `nav` that you already have.

File: src/components/Navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <div className={styles.logo}>
+        <Logo />
+      </div>
+      <nav className={styles.nav}>
+        <div className={styles.link}>
```
Comment (jfranciscosousa) [inline src/components/Navbar/index.js]: Usually, navs go like this `nav > ul > li > a`. Why? It's a pattern used for accessibility. Usually navs are a list of links and users of screen readers are usually expecting lists with links on the navigation part.

Also, every time we have repetition of stuff, `ul` is a good options.

**Review by jfranciscosousa — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.js (hunk 2 above)
Comment (DavideSilva) [reply src/components/Navbar/index.js]: `NavLink` is already an `a`, so I simply switched the `div`s for `li`s and surrounded that with an `ul`

File: src/components/Navbar/index.js (hunk 1 above)
Comment (DavideSilva) [reply src/components/Navbar/index.js]: Didn't knew that. Switched for a `div` :+1:

**Review by jfranciscosousa — APPROVED** [review]
Impecable

**Pull request merged**